x/crisis/spec/02_messages.md

```diff
@@ -11,10 +11,12 @@ corresponding updates to the state.
 
 Blockchain invariants can be checked using the `MsgVerifyInvariant` message. 
 
-```go
-type MsgVerifyInvariant struct {
-	Sender         sdk.AccAddress 
-	InvariantRoute string
+```protobuf
+// MsgVerifyInvariant represents a message to verify a particular invariance.
```

Comment:
we could use the `+++` syntax here too, see https://github.com/cosmos/cosmos-sdk/pull/8279/files#r553882601